easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -766,6 +766,14 @@ def remove_false_versions(deps):
                 builddeps = [self._parse_dependency(dep, build_only=True) for dep in builddeps]
             self['builddependencies'] = remove_false_versions(builddeps)
 
+            sourcedeps = self['source_deps']
```

Comment:
@PetrKralCZ Let's avoid confusion and use `source_deps` consistently (more changes needed below):
```suggestion
            source_deps = self['source_deps']